config/remote-config-prod.json

```diff
@@ -1,5 +1,28 @@
 {
-	"country_codes_to_region_codes": [{"countryCode":"AU","regionCodes":["my"]},{"countryCode":"CN","regionCodes":["my"]},{"countryCode":"HK","regionCodes":["my"]},{"countryCode":"IN","regionCodes":["my"]},{"countryCode":"ID","regionCodes":["my"]},{"countryCode":"JP","regionCodes":["my"]},{"countryCode":"PH","regionCodes":["my"]},{"countryCode":"KR","regionCodes":["my"]},{"countryCode":"TH","regionCodes":["my"]},{"countryCode":"VN","regionCodes":["my"]},{"countryCode":"SG","regionCodes":["sg"]},{"countryCode":"US","regionCodes":["us"]},{"countryCode":"MY","regionCodes":["my"]},{"countryCode":"NO","regionCodes":["no"]}],
-	"region_groups": [{"name":"Asia","description":"Southeast asia & pacific","backgroundColor":"lipstick","isPreview":false,"countryCodes":["SG","MY"]},{"name":"The Americas","description":"Latin & north america","backgroundColor":"azul","isPreview":true,"countryCodes":[]},{"name":"Loltel","description":"Only certain people should see this","backgroundColor":"yellow","isPreview":true,"countryCodes":["NO"]}]
-	"feature_flags": {"enableStripeInsteadOfApplePay":false}
+  "parameters": {
+    "feature_flags": {
+      "defaultValue": {
+        "value": "{\"enableStripeInsteadOfApplePay\":false}"
+      }
+    },
+    "region_groups": {
+      "defaultValue": {
+        "value": "[{\"name\":\"Asia\",\"description\":\"Southeast asia & pacific\",\"backgroundColor\":\"lipstick\",\"isPreview\":false,\"countryCodes\":[\"SG\",\"MY\"]},{\"name\":\"The Americas\",\"description\":\"Latin & north america\",\"backgroundColor\":\"azul\",\"isPreview\":true,\"countryCodes\":[]},{\"name\":\"Loltel\",\"description\":\"Only certain people should see this\",\"backgroundColor\":\"yellow\",\"isPreview\":false,\"countryCodes\":[\"NO\"]}]"
+      }
+    },
+    "country_code_to_region_codes": {
+      "defaultValue": {
+        "value": "[{\"countryCode\":\"AU\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"CN\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"HK\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"IN\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"ID\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"JP\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"PH\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"KR\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"TH\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"VN\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"SG\",\"regionCodes\":[\"sg\"]},{\"countryCode\":\"US\",\"regionCodes\":[\"us\"]},{\"countryCode\":\"MY\",\"regionCodes\":[\"my\"]},{\"countryCode\":\"NO\",\"regionCodes\":[\"no\"]}]"
+      }
+    }
+  },
+  "version": {
+    "versionNumber": "3",
+    "updateTime": "2019-10-29T13:07:48.179Z",
+    "updateUser": {
+      "email": "havard@redotter.sg"
```

Comment:
Could this somehow be substituted in from somewhere more prominent?  Adding personal email addresses is in general a discouraged practice, but when it has to be done, it should be done in a way that is prominent, and easy to modify when the opportunity arises.

Reply:
I agree, this email most likely came from when I changed the remote config manually through firebase console. If we change it using this script, the correct email should be the `client_email` inside service-account.json for production.  I've updated the email accordingly.